towhee/runtime/operator_manager/operator_action.py

```diff
@@ -92,6 +96,20 @@ def serialize(self):
                 'tag': self._tag
             }
         elif self._type == 'lambda':
-            raise ValueError('Lambda not supported yet.')
+            self._loaded_fn = marshal.dumps(self._fn.__code__)
+            return {
+                'operator': self._loaded_fn,
+                'type': self._type,
+                'init_args': None,
+                'init_kws': None,
+                'tag': None
+            }
         elif self._type == 'callable':
-            raise ValueError('Callable not supported yet.')
+            self._loaded_fn = pickle.dumps(self._fn)
```

Comment:
Can the dag dump into a JSON file? if not, in the local model, we just store the func itself

Reply:
Done, return the func.